docs/design.ltx

```diff
@@ -0,0 +1,337 @@
+\IEEEpeerreviewmaketitle
+
+\newpage
+
+\tableofcontents
+
+\newpage
+
+\section{Introduction}
+\label{sec:intro}
+This is the design document for COAL. It outlines the design of our system as well as the research and development priorities at each step. The COAL suite of algorithms constitutes a pipeline of data processing and presentation, and is accompanied by API documentation as well as a website for end users and the general public. Specific implementation details remain to be determined based on research, but in general the program reduces a large set of hyperspectral and geographic data into comprehensible conclusions about mining and environmental impact in human and machine-friendly formats.
+
+Each section was authored by the corresponding team member responsible for each component identified in the technology review: Sections \ref{subsec:featureextraction} and \ref{subsec:classification} by Heidi; section \ref{subsec:identifymining} by Xiaomei; sections \ref{subsec:preprocessing}, \ref{subsec:correlate}, and \ref{subsec:changesovertime} by Taylor; section \ref{subsec:apidoc} by Heidi; and sections \ref{subsec:staticsitegenerator} and \ref{subsec:frontendframework} by Xiaomei. The research priorities and design choices for each part of the project are discussed in each section.
+
+\section{Technologies}
+
+% heidi: requirement 3.4.1 part 1: choose technology for mineral identification and classification
+\subsection{Feature extraction in AVIRIS data}
+\label{subsec:featureextraction}
+
+In order to process hyperspectral data, machine learning techniques will have to be used to process the raw AVIRIS images \cite{aviris}. Further processing could be done using blob detection, as described below.
+
+AVIRIS images cover a lot of land mass with a lot of different features, so a method of feature extraction is needed to automatically pick out the different regions on the image. In blob detection, an area of an image that is different from the surrounding pixels is known as a blob. This will make classification easier as instead of needing to classify every single pixel every single time, the classification algorithm will only need to process each blob, or “feature,” which typically contains many pixels. The blob detection algorithm that will be used is automatic scale detection. It is one of the more expensive methods of blob detection, but it is highly accurate, so it is the one that has been chosen as discussed in the technology review. The automatic scale detection algorithm is based on the Laplacian of Gaussian. As the name suggests, this is a convolution of the Laplacian and Gaussian filters \cite{autoscaleprinc}. The Gaussian filter is applied first to smooth out noise and then the Laplacian filter is used to detect the shapes in the image \cite{lapofgaus}. The equation for a Gaussian convolution is given below, where t represents scale \cite{autoscaleprinc}.
+
+\[ g(x, y) = \frac{1}{2\pi t}e^{-\frac{(x^{2} + y^{2})}{2t}}\] 
```

Comment:
I'm not familiar with the paper but I would assume that this is _the_ `e`, [Euler's number](https://en.wikipedia.org/wiki/E_%28mathematical_constant%29).